pettingzoo/utils/conversions.py

```diff
@@ -433,7 +451,7 @@ def reset(self, seed=None, options=None):
             if not (self.aec_env.terminations[agent] or self.aec_env.truncations[agent])
         }
 
-        infos = dict(**self.aec_env.infos)
+        infos = {**self.aec_env.infos}
```

Comment:
It may be more readable to do dict() in this case, don't see a reason to change it to be honest

Reply:
Actually, using `dict()` doesn't work, since it requires keys that are strings, which is not necessarily the case now. However, `{}` does work for non-string keys.

Reply:
Oh interesting, good catch then